shell/src/main/java/alluxio/cli/fs/command/RmCommand.java

```diff
@@ -122,15 +132,15 @@ public int run(CommandLine cl) throws AlluxioException, IOException {
 
   @Override
   public String getUsage() {
-    return "rm [-R/-r/--recursive] [-U] [--alluxioOnly] <path>";
+    return "rm [-R/-r/--recursive] [-U] [--alluxioOnly] [-m/--deleteMountPoint] <path>";
   }
 
   @Override
   public String getDescription() {
     return "Removes the specified file. Specify -R/-r/--recursive to remove file"
         + " or directory recursively. Specify -U to remove directories without checking "
-        + " UFS contents are in sync. Specify -alluxioOnly to remove data and metadata from"
-        + " alluxio space only.";
+        + " UFS contents are in sync. Specify --alluxioOnly to remove data and metadata from"
+        + " alluxio space only. Specify -m/--deleteMountPoint to allow removing mount points";
```

Comment:
```suggestion
        + " alluxio space only. Specify -m/--deleteMountPoint to allow removing mount points, "
        +"otherwise the rm will fail if the directory contains mount points.";
```